resources/lib/services/msl/request_builder.py

```diff
@@ -72,16 +73,21 @@ def _headerdata(self, esn, is_handshake=False, is_key_request=False,
             'handshake': is_handshake,
             'nonreplayable': False,
             'capabilities': {
+<<<<<<< HEAD
```

Comment:
This file as well.

Reply:
thx for your sugesstions.
I have fixed them all allready